src/OmniSharp.Roslyn.CSharp/Services/Refactoring/V2/CodeFixesForProjects.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Composition;
+using System.Linq;
+using System.Reflection;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using OmniSharp.Services;
+using OmniSharp.Utilities;
+
+namespace OmniSharp.Roslyn.CSharp.Services.Refactoring.V2
+{
+    [Shared]
+    [Export(typeof(CodeFixesForProjects))]
+    public class CodeFixesForProjects
+    {
+        private readonly ConcurrentDictionary<string, IEnumerable<CodeFixProvider>> _codeFixCache = new ConcurrentDictionary<string, IEnumerable<CodeFixProvider>>();
+        private readonly IAssemblyLoader _assemblyLoader;
```

Comment:
why this has assemblyLoader? assemblyLoader is supposed to be only used for AnalyzerFileReference...

Reply:
I guess it looks strange to me since I am used to our implementation. there is nothing wrong that you use analyzer assembly loader here especially for you, it is not just analyzer loader but also assembly loader.